platform-hub-web/src/app/kubernetes-tokens/kubernetes-tokens-list.component.js

```diff
@@ -0,0 +1,100 @@
+export const KubernetesTokensListComponent = {
+  template: require('./kubernetes-tokens-list.html'),
+  controller: KubernetesTokensListController
+};
+
+function KubernetesTokensListController(roleCheckerService, hubApiService, logger, $stateParams, $mdDialog) {
+  'ngInject';
+
+  const ctrl = this;
+  const userId = $stateParams.userId;
+
+  ctrl.searchSelectedUser = null;
+  ctrl.loading = true;
+  ctrl.isAdmin = false;
```

Comment:
I think this entire component is guaranteed to only be routed to if you're an admin (assuming you agree with my comment above in the routes). In which case it doesn't need to be aware of admin status at all and can assume it (should hopefully simplify things).